### ISSUE TITLE
How does the night sky (the universe) vary on human timescales?

### ISSUE THREAD

**Issue by SimonStevenson** [ethrane/transients#6]
Many of the questions that this working group is interested in are underpinned by transient surveys, searching for variability in a given messenger/wavelength on a given timescale. Whether that is searching for emission from FRBs at other wavelengths, finding novel radio transients, finding new classes of supernovae etc. Rubin/LSST is going to be the ultimate version of this in the optical. Sure, you can use these things to probe physics, but ultimately, we just want to know what goes bump in the night.

**Comment by btucker22** [comment]
And this links in with Aboriginal and Torres Strait Islander Astronomy what records / knowledge may exist that we can link to new/existing/discovered transients

**Comment by timbedding** [comment]
Yes, and even changes that happen very slowly might be measurable with careful/clever methods.  Examples include changes inside stars as they burn through their fuel, changes in binary stars as the components evolve, etc.

**Comment by clidman** [comment]
This is a very broad question, but given the last Decadal Plan only had 6 key questions, perhaps we need a question like this to cover all the great science we do.

**Comment by cwjames1983** [comment]
I'd like to add that from an FRB perspective, I think it'd make more sense to have them fit in this category than in the "how do stars die" category, though "extreme matter" could also work. This is extremely instrumentation-driven. Having a question like this would be great. Fully agree with @clidman here.

**Comment by cwjames1983** [comment]
And @SimonStevenson 's final sentence of "but ultimately, we just want to know what goes bump in the night." is really the best description for this WG I've heard!

**Comment by jbroderick1** [comment]
I like this question; as part of SKA Science Operations planning, we're already thinking a lot about how to maximise commensal transient searches.

**Comment by ethrane** [comment]
We discussed this a bit on the call, but I wanted to record my thoughts here as well. I think this key question is phrased too broadly for what we need these key questions to be: a tool for prioritising infrastructure investments. It would be nice if we could support time-domain astronomy as a discipline, but advocating for everything is functionally equivalent to advocating for nothing. So while I agree the text is a nice description of the broad field, I think we need to ask questions that identify more specific areas of investment.

**Comment by timbedding** [comment]
Many thanks for organising the discussion.  One comment from me: it seems that most people on the zoom considered "Time-domain astronomy" (as in the title of this working group) to mean transients, rather than the broader topic of things that vary in the sky.  My recollection of the NCA discussions about the addition of this new group were that it was intended to include more than just transients (I am no longer on the NCA, but you might want to check).  Hence, it would also include things like exoplanet studies, asteroseismology, variable AGN, etc.  One advantage of doing so is that it brings the ability to emphasise Australia's longitude as an important factor, and also includes international facilities like LSST, TESS, etc.

**Comment by ethrane** [comment]
@timbedding, you are right that our group's scope is time-domain astronomy––and also multi-messenger astronomy! So we cover a lot. However, it's worth keeping in mind how we fit into the big picture. For example, there's a dedicated group (1.2 Stars, Planets, and the Galaxy), which I believe is the natural home for exoplanets. I think our group will be most effective if we use identify key questions that are not just supporting the core activities of other groups. Synergy with other groups is good, but I'd like to focus on what the time-domain astronomy community brings to the table that is special. And, in any case, we have to prioritise.

**Comment by bersavosh** [comment]
I really like this question, and my only comment is to encourage a wording that perhaps encapsulates more physics. 

Also @ethrane I disagree with your thoughts here. You pitch breadth and synergy as a positive [in Q2](https://github.com/ethrane/transients/issues/2#issue-2129414550) and then in this question you are using the same impact as a negative.

**Comment by ethrane** [comment]
@bersavosh, I think https://github.com/ethrane/transients/issues/2 is pretty focused: it's about black holes. I'm excited about the synergies with the Galaxies & Cosmology, but I don't think people will see that question and think: "this question pertains to just about everything!" This question, on the other hand, covers the entire field of transient astronomy. We can put this question into our report, but I worry the next stage of editors will read it and think, "the time-domain group supports time-domain astronomy as their priority?" I think we can have a greater influence with other questions.

**Comment by Ch-Wolf** [comment]
❤️

**Comment by jeffcooke** [comment]
Hi all. As you may have seen from discussion in Key Question #11 'How do transients inform us about the Universe?", we wanted to get thoughts from the full WG on whether it would be better if “How does the night sky (the universe) vary on human timescales” and “How do transients inform us about the Universe” are separate questions or whether to combine them into one and change the phrasing. Please see the description in #11 to assess what that question is asking to assess this. 

It's been stated that we need broad questions to be inclusive but, perhaps, at the same time a bit more specific questions on the aims. To summarise (but, apologies, I'll not do either question justice!), "How does the night sky (the Universe) vary on human timescales?" roughly supports search and discovery of transients, new phenomena, etc. (on human timescales) and, importantly, variables to know what goes 'bump in the night'. "How do to transients inform us about the Universe? aims to support work done with transients and often on cosmic timescales, that help us understand the Universe, from the expansion rate of the Universe, absorption-line, ionised baryon content, star formation rate research over cosmic time and other research areas informed exclusively by transients and from the transient perspective, i.e., the observations, follow-up, theory, physics, etc. of transients necessary to achieve this understanding.

**Comment by taramurphy** [comment]
I'm just joining this discussion, but I think this question is what I'd consider "descriptive". Ideally, I think, key science questions should identify a gap in our understanding of Universe (and/or specifically physics) that we don't know the answer to, rather than a question for which the answer would be an description of the natural world. 

Observations (such as characterising how the sky changes) are then necessary to address the key question.
